Source/Lib/Codec/EbPictureControlSet.h

```diff
@@ -193,6 +193,7 @@ typedef struct SbParameters
     uint8_t   is_complete_sb;
     EB_BOOL   pa_raster_scan_block_validity[PA_BLOCK_MAX_COUNT];
     EB_BOOL   ep_scan_block_validity[EP_BLOCK_MAX_COUNT];
+    uint32_t  ep_scan_block_valid_block[EP_BLOCK_MAX_COUNT];
```

Comment:
It might be better to move this to the end of the struct to prevent unnecessary padding

Reply:
@1480c1 - not sure if it would make any difference give the current layout of the structure which seems unoptimized.
Also the structure sizes are off sizes: EP_BLOCK_MAX_COUNT is an odd size = 681 and PA_BLOCK_MAX_COUNT=85.